lib/buttons-and-links/KButton.vue

```diff
@@ -108,13 +108,7 @@
     },
     computed: {
       iconColor() {
-        if (this.primary) {
-          return this.appearance === 'raised-button'
-            ? this.$themeTokens.textInverted
-            : this.$themeTokens.primary;
-        } else {
-          return this.$themeTokens.text;
-        }
+        return this.primary ? this.$themeTokens.primaryDark : this.$themeTokens.primary;
```

Comment:
Because this can be a link (as in <a> tag) or a button we need to account for whether the `appearance` is `raised-button` - in which case, the text and icon colors should be `$themeTokens.textInverted`.